lentille-github/src/Lentille/GitHub/Issues.hs

```diff
@@ -159,7 +160,11 @@ transformResponse searchResult =
     extractUrl item = case item of
       Just
         ( SearchNodesTimelineItemsNodesConnectedEvent
-            (SearchNodesTimelineItemsNodesSubjectPullRequest (url))
-          ) -> show url
```

Comment:
The show implementation for Text does adds quotes. I don't think we need to further decode the value and unwrapping the URI type may be enough:

  (SearchNodesTimelineItemsNodesSubjectPullRequest (URI url))) -> url